packages/keycloak-user-management/src/components/forms/UserForm/tests/utils.test.tsx

```diff
@@ -180,4 +223,49 @@ describe('forms/utils/submitForm', () => {
     expect(notificationErrorMock).toHaveBeenCalledWith(ERROR_OCCURED);
     expect(historyPushMock).not.toHaveBeenCalled();
   });
+
+  it('marks user as practitioner successfully', async () => {
+    const valuesCopy = {
+      ...values,
+      active: true,
+      id: keycloakUser.id,
+    };
+
+    createOrEditPractitioners(
+      accessToken,
+      opensrpBaseURL,
+      opensrpServiceClass,
+      valuesCopy,
+      undefined,
+      false
+    );
+
+    await act(async () => {
+      await flushPromises();
+    });
+    expect(notificationSuccessMock).toHaveBeenCalledWith('Practitioner created successfully');
```

Comment:
Have we added a test to assert **Practioner updated successfully**?

Reply:
@kelvin-muchiri addressed via https://github.com/OpenSRP/web/pull/196/commits/5d0a31ed0084ef1d494a1ac36772ace24839acc1